.k8s/dev/ingress.yaml

```diff
@@ -3,6 +3,13 @@ kind: Ingress
 metadata:
   name: laa-court-data-ui-app-ingress
   namespace: laa-court-data-ui-dev
+  annotations:
+  nginx.ingress.kubernetes.io/server-snippet: |-
```

Comment:
Is the tabbing correct here? All the others are tabbed in more.

Reply:
Fixed